### PR TITLE
Clean up anchor-positioning tests.

### PR DESCRIPTION
**Pull request by jgerigmeyer**

### CODE REVIEW

File: css/css-anchor-position/anchor-position-dynamic-002.html [hunk 1]
```diff
@@ -41,14 +41,14 @@
...
+checkLayout('.target', false);
 container.classList.add('after');
 for (const element of document.getElementsByClassName('target')) {
   if (element.dataset.offsetX === '5')
     element.dataset.offsetX = '10';
   if (element.dataset.expectedWidth === '5')
     element.dataset.expectedWidth = '10';
 }
-checkLayout('.after .target');
+checkLayout('.after .target', true);
```
Comment (jgerigmeyer) [inline css/css-anchor-position/anchor-position-dynamic-002.html]: This follows the same pattern as other tests (e.g. `css-grid`) where `checkLayout()` is called multiple times, with changes in between.

File: css/css-anchor-position/anchor-position-dynamic-004.html [hunk 2]
```diff
@@ -39,9 +39,7 @@
          data-expected-width=30 data-expected-height=20></div>
   </div>
 <script>
-function run_test() {
-  document.body.classList.add('after');
-  checkLayout('.target');
-}
+document.body.classList.add('after');
+checkLayout('.target');
```
Comment (jgerigmeyer) [inline css/css-anchor-position/anchor-position-dynamic-004.html]: This brings this test in line with the pattern used in the other `anchor-position-dynamic` tests.

File: css/css-anchor-position/anchor-query-fallback.html [hunk 3]
```diff
@@ -34,34 +34,36 @@
...
-  <div class="flex-item"></div>
-  <div class="flex-item"></div>
-  <div class="flex-item"></div>
-  <div class="flex-item"></div>
-  <div class="flex-item"></div>
-  <div class="flex-item"></div>
-  <div class="flex-item"></div>
-  <div class="flex-item" id="a2"></div>
+<body onload="checkLayout('.target')">
+  <div id="container">
```
Comment (jgerigmeyer) [inline css/css-anchor-position/anchor-query-fallback.html]: Added missing `div` container.

File: css/css-anchor-position/anchor-query-fallback.html [hunk 4]
```diff
@@ -34,34 +34,36 @@
...
-  <div class="target" style="top: anchor(--a1 left, calc((anchor(--a1 bottom) + anchor(--a2 top)) / 2)" data-offset-y="75"></div>
-  <div class="target" style="width: anchor-size(--inexist-anchor width, 50%)" data-expected-width="150"></div>
-  <div class="target" style="width: anchor-size(--inexist-anchor width, calc(20% + 20px))" data-expected-width="80"></div>
-  <div class="target" style="height: anchor-size(--inexist-anchor height, anchor-size(--a1 width))" data-expected-height="100"></div>
-  <div class="target" style="height: anchor-size(--inexist-anchor height, calc((anchor-size(--a1 width) + anchor-size(--a2 height)) / 2)" data-expected-height="75"></div>
+    <!-- More complicated fallback values -->
+    <div class="target" style="left: anchor(--inexist-anchor left, 50%)" data-offset-x="150"></div>
+    <div class="target" style="left: anchor(--inexist-anchor left, calc(20% + 20px))" data-offset-x="80"></div>
+    <div class="target" style="top: anchor(--a1 left, anchor(--a2 top))" data-offset-y="100"></div>
+    <div class="target" style="top: anchor(--a1 left, calc((anchor(--a1 bottom) + anchor(--a2 top)) / 2))" data-offset-y="75"></div>
```
Comment (jgerigmeyer) [inline css/css-anchor-position/anchor-query-fallback.html]: Fixed syntax error: added missing `)` at end of anchor fn.

File: css/css-anchor-position/anchor-query-fallback.html [hunk 5]
```diff
@@ -34,34 +34,36 @@
...
+    <div class="target" style="width: anchor-size(--inexist-anchor width, 50%)" data-expected-width="150"></div>
+    <div class="target" style="width: anchor-size(--inexist-anchor width, calc(20% + 20px))" data-expected-width="80"></div>
+    <div class="target" style="height: anchor-size(--inexist-anchor height, anchor-size(--a1 width))" data-expected-height="100"></div>
+    <div class="target" style="height: anchor-size(--inexist-anchor height, calc((anchor-size(--a1 width) + anchor-size(--a2 height)) / 2))" data-expected-height="75"></div>
```
Comment (jgerigmeyer) [inline css/css-anchor-position/anchor-query-fallback.html]: Fixed syntax error: added missing `)` at end of anchor fn.

**Comment by mfreed7** [comment]
This was landed in Chromium, and the corresponding WPT pull should land soon:

https://github.com/web-platform-tests/wpt/pull/38224

**Comment by jgerigmeyer** [comment]
Merged in https://github.com/web-platform-tests/wpt/pull/38224

**Pull request closed without merging**